packages/trees/frontend/sirius-components-trees/src/treeitems/TreeItemContextMenu.tsx

```diff
@@ -235,6 +242,7 @@ export const TreeItemContextMenu = ({
       invokeSingleClick(menuEntry.id);
       onClose();
     }
+    // Do not handle ContributedTreeItemContextMenuEntry, it is done in the contributed component.
```

Comment:
that's not the concern of this component

Reply:
I removed the comment